src/expr.rs

```diff
@@ -458,7 +458,51 @@ impl Expr {
         SimpleExpr::Custom(s.to_owned())
     }
 
-    /// Express any custom expression with [`Value`]. Use this if your expression needs variables.
+    #[deprecated(since = "0.28.0", note = "Please use the [`Expr::cust_with_expr`]")]
+    pub fn cust_with_values<V, I>(s: &str, v: I) -> SimpleExpr
+    where
+        V: Into<SimpleExpr>,
+        I: IntoIterator<Item = V>,
+    {
+        SimpleExpr::CustomWithExpr(s.to_owned(), v.into_iter().map(|v| v.into()).collect())
+    }
```

Comment:
Hey @ikrivosheev, it's still being deprecated. Seems you didn't push the changes to GitHub? hahaaa

Reply:
@billy1624 sorry, I don't understand... I have already deprecated `cust_with_values`. What I need to do?)

Reply:
I think @tyt2y3 want to keep it instead of deprecating it

Reply:
Why we need this function? `cust_with_exprs` - can do all the same

Reply:
It seems cleaner to be able to:

```rust
.and_where(Expr::cust_with_values("6 = $2 * $1", [3, 2]))
```

without `.into()`